src/controllers/profiles.js

```diff
@@ -0,0 +1,10 @@
+const ErrorResponse = require('../utils/errorResponse');
+const User = require('../models/user');
+
+exports.getAllProfiles = async (req, res, next) => {
```

Comment:
As Muslim mentioned, `getAllProfiles` should be changed to `getMyProfile` since this will return only the user itself.